taskt/UI/Forms/ThemedForm.cs

```diff
@@ -26,27 +22,24 @@ private void ThemedForm_Load(object sender, EventArgs e)
         protected override void OnShown(EventArgs e)
         {
             base.OnShown(e);
-            if (this.ClientSize.Height > Screen.PrimaryScreen.WorkingArea.Height - this.CurrentAutoScaleDimensions.Height) // Resizes if too tall to fit
+            // Resizes if too tall to fit
+            if (ClientSize.Height > Screen.PrimaryScreen.WorkingArea.Height - CurrentAutoScaleDimensions.Height)
             {
-                this.ClientSize = new Size(this.ClientSize.Width, Screen.PrimaryScreen.WorkingArea.Height - (int)this.CurrentAutoScaleDimensions.Height);
+                ClientSize = new Size(ClientSize.Width, Screen.PrimaryScreen.WorkingArea.Height -
+                    (int)CurrentAutoScaleDimensions.Height);
             }
         }
 
-        private taskt.Core.Theme _Theme = new taskt.Core.Theme();
-        public taskt.Core.Theme Theme
-        {
-            get { return _Theme; }
-            set { _Theme = value; }
-        }
-
```

Comment:
Was there a particular reason for previous implementation vs full public swap?

Reply:
I couldn't find one which is why I swapped it. Maybe there was an intended purpose at the time but it was never used. Only Theme is ever referenced. _Theme is never used for anything other than the get/set